samples/packages/pulumi-typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@openfabr/package-ri-pulumi",
-    "version": "1.0.0",
+    "version": "1.0.1",
```

Comment:
You don't need to do that as it should be left for the release tooling to decide. However you do need to follow the conventional commit format, and in this case I'd suggest `fix(package-ri-pulumi): X' to make the bump.